### PR TITLE
modules/darwin: authorizedKeys updates

### PR DESCRIPTION
**Pull request by zowoq**

### CODE REVIEW

File: modules/darwin/common/default.nix [hunk 1]
```diff
@@ -58,15 +57,11 @@ in
   # disable application layer firewall, telegraf needs an incoming connection
   system.defaults.alf.globalstate = 0;
 
-  # srvos
-  services.openssh.authorizedKeysFiles = pkgs.lib.mkForce [ "/etc/ssh/authorized_keys.d/%u" ];
-
   # srvos
   environment.etc."ssh/sshd_config.d/darwin.conf".text = ''
     HostKey /etc/ssh/ssh_host_ed25519_key
```
Comment (zowoq) [inline modules/darwin/common/default.nix:62]:
```suggestion
  environment.etc."ssh/sshd_config.d/darwin.conf".text = ''
    AuthorizedKeysFile none
    HostKey /etc/ssh/ssh_host_ed25519_key
```

**Comment by emilazy** [comment]
Just a heads up: https://github.com/LnL7/nix-darwin/pull/976 has now been merged, so you shouldn’t need to use the fork here and the next upgrade would start complaining about `/etc/authorized_keys.d` anyway because of the security warning :)

File: modules/darwin/common/default.nix (hunk 1 above)
Comment (zowoq) [reply modules/darwin/common/default.nix]: We may need to reinstall or switch the community builder to a different machine so easier to just require that the keys are managed.

Comment (emilazy) [reply modules/darwin/common/default.nix]: Makes sense, I forgot about the warning about `$HOME` :)

**Pull request merged**